payloads/v9/channel.ts

```diff
@@ -79,37 +78,45 @@ export interface APIChannel extends APIPartialChannel {
 	 * The absence of this field in API calls and Gateway events should indicate that slowmode has been reset to the default value.
 	 */
 	rate_limit_per_user?: number;
+}
+
+export interface APITextChannel extends APITextBasedChannel {
+	type: ChannelType.GuildText;
 	/**
-	 * The recipients of the DM
-	 *
-	 * See https://discord.com/developers/docs/resources/user#user-object
+	 * Default duration for newly created threads, in minutes, to automatically archive the thread after recent activity
 	 */
-	recipients?: APIUser[];
+	default_auto_archive_duration?: ThreadAutoArchiveDuration;
 	/**
-	 * Icon hash
+	 * The channel topic (0-1024 characters)
 	 */
-	icon?: string | null;
+	topic?: string | null;
+}
+
+export interface APINonRateLimitedChannel extends Omit<APITextChannel, 'rate_limit_per_user' | 'type'> {
+	type: ChannelType.GuildNews;
+}
+
+export interface APINonInteractiveChannel extends BaseAPIChannel {
+	type: ChannelType.GuildCategory | ChannelType.GuildStore;
+}
+
+export interface APIOwnedChannel extends APITextBasedChannel {
 	/**
 	 * ID of the DM creator or thread creator
 	 */
 	owner_id?: Snowflake;
+}
+
+export interface APIVoiceChannel extends BaseAPIChannel {
+	type: ChannelType.GuildVoice | ChannelType.GuildStageVoice;
```

Comment:
These two channels should be separated into APIGuildVoiceChannel and APIGuildStageVoiceChannel, where the latter is an alias to the former